pkgs/tools/misc/spigot/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, buildPackages
+, fetchurl
+, gmp
+}:
+
+stdenv.mkDerivation rec {
+  name = "spigot-20200101";
+  src = fetchurl {
+    url = "https://www.chiark.greenend.org.uk/~sgtatham/spigot/${name}.b1b0b20.tar.gz";
+    sha256 = "0cf9h6x2n016ngy5qx53kkmgi4v8a3zinmh534j4gafmmzap061s";
+  };
+
+  configureFlags = [ "--with-gmp" ];
+
+  depsBuildBuild = [ buildPackages.stdenv.cc ];
+  nativeBuildInputs = [
+    # Tools
+    buildPackages.gmp
+  ];
+  buildInputs = [ gmp ];
+
+  doCheck = true; # not cross
```

Comment:
What does the comment mean? If you're referring to cross-compilation, then that is a standard issue and not specific to this package.